ui/sdk/src/hamilton_sdk/adapters.py

```diff
@@ -45,11 +45,12 @@ def __init__(
         dag_name: str,
         tags: Dict[str, str] = None,
         client_factory: Callable[
-            [str, str, str], clients.HamiltonClient
+            [str, str, str, str | bool], clients.HamiltonClient
```

Comment:
this won't work for python < 3.10, right?

So we either import futures, or we use `Union[str, bool]`, right?

Reply:
nvm looking at old version.